examples/set_server_boot_once.psm1

```diff
@@ -173,7 +172,7 @@ function set_server_boot_once
     # Delete existing session whether script exit successfully or not
     finally
     {
-        if ($session_key -ne "")
+        if ([string]::IsNullOrWhiteSpace($session_key))
```

Comment:
@ClaudioG64 Many thanks for your contribution. Good enhancement to code! Only one point:
if ([string]::IsNullOrWhiteSpace($session_key)) should be changed to if (-Not ...) because only when session key exist then delete sessions.  Could you change this and submit again? thanks!

Reply:
Oh yes ! Good catch :-)
It was a test to work even in the session is $Null
Thanks

Reply:
Thank you very much for your contribution!